optimum/onnxruntime/subpackage/commands/quantize.py

```diff
@@ -17,7 +17,7 @@
 from pathlib import Path
 from typing import TYPE_CHECKING
 
-from .. import BaseOptimumCLICommand
+from optimum.commands import BaseOptimumCLICommand
```

Comment:
why absolute path?

Reply:
That's mostly because the relative path would have been complicated, but also to serve as a model for actual subpackages that can live outside the optimum tree.